docs/architecture/blazor-for-web-forms-developers/app-startup.md

```diff
@@ -1,15 +1,108 @@
+
+The `Configure` method introduces the concept of the HTTP pipeline to ASP.NET Core.  In this method, we declare from top to bottom the methods, called [Middleware](middleware.md), that will handle every request sent to our application.  Most of these features in the default configuration were scattered across the web forms configuration files and are now in one place for ease of reference.
+
+No longer is the configuration of the custom error page placed in a `web.config` file, but now is configured to always be shown if the application environment is not labeled `Development`.  Additionally, ASP.NET Core applications are now configured to serve secure pages with TLS by default with the `UseHttpsRedirection` method call.
+
+Next, an unexpected configuration method is listed to `UseStaticFiles`.  In ASP.NET Core, files that are not interpreted by .NET like JavaScript, CSS, images, and other content need to be explicitly configured to be accessible by visitors to our application.
+
+The next line is the first that replicates one of the configuration options from web forms: `UseRouting`.  This method adds the ASP.NET Core router to the pipeline and it can be either configured here or in the individual files that it can consider routing to.  More information about routing configuration can be found in the [Routing section](page-routing-layouts.md).
+
+The final statement in this method defines the endpoints that ASP.NET Core is listening on.  These are the web accessible locations that you can access on the web server and receive some content handled by .NET and returned to you.  The first entry, `MapBlazorHub` configures a SignalR hub for use in providing the real-time and persistent connection to the server where the state and rendering of Blazor components will take place.  The `MapFallbackToPage` method call indicates the web-accessible location of the page that starts the Blazor application and also configures the application to handle deep-linking requests from the client-side.  You will see this feature at work if you open a browser and navigate directly to your application and a Blazor handled route, such as `/counter` in the default project template.
+
+## Upgrading the BundleConfig Process
+
+We recognize that bundling assets like CSS stylesheets and JavaScript files has evolved significantly, with other technologies providing quickly evolving tools and techniques for managing these resources.  To this end, there are several techniques recommended for managing these resources:
+
+- Use an Node command-line tool such as Grunt / Gulp / WebPack
+- Use the .NET command-line `bundle` tool with `bundleconfig.json` configuration file
```

Comment:
We no longer recommend this tool in our latest guidance.

Reply:
Removed reference to the bundle tool and reworded the prior sentence